README.md

```diff
@@ -26,3 +26,20 @@ This list should not be expected to be comprehensive nor exclusive, and may be c
 ## Request a tweet from the [@nodejs](https://twitter.com/nodejs) Twitter account
 
 To request a tweet from the @nodejs account, follow the instructions here: [/tweets](./tweets/)
+
+## Process for tweets regarding Node.js releases:
+
+For all major and minor releases, the following process should be done:
+
+1. Retweet canonical tweet from person leading the effort
+1. Quote tweet the canonical tweet with the following language:
```

Comment:
maybe specify a delay here to give the original tweet some time to get organic exposure, be referred to as the original source of info, etc

I'm not sure on what the delay should be though, maybe 24h ? 12h ?

Reply:
Yeah, its a matter of balancing credit and awareness. I thought that quote tweeting satisfied both but am fully open to tweaks or different approaches.